packages/concerto-core/lib/decoratormanager.js

```diff
@@ -440,15 +446,35 @@ class DecoratorManager {
         }
     }
 
+    /**
+     * Executes a Command against a Model Namespace, adding
+     * decorators to the Namespace.
+     * @param {*} model the model
+     * @param {*} command the Command object from the dcs
+     * org.accordproject.decoratorcommands model
+     */
+    static executeNamespaceCommand(model, command) {
```

Comment:
This should be a private function, I believe?

Reply:
Corrected